plugins/modules/vmware_cluster_dpm.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Swisscom (Schweiz) AG
+# Author(s): Olivia Luetolf <olivia.luetolf@swisscom.com>
+# Copyright: (c) 2018, Ansible Project
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: vmware_cluster_dpm
+short_description: Manage Distributed Power Management (DPM) on VMware vSphere clusters
+description:
+    - Manages DPM on VMware vSphere clusters.
+    - All values and VMware object names are case sensitive.
+author:
+- Olivia Luetolf (@olilu)
+requirements:
+    - Tested on ESXi 6.7 and 7.0.
+    - PyVmomi installed.
+options:
+    cluster_name:
+      description:
+      - The name of the cluster to be managed.
+      type: str
+      required: true
+    datacenter:
+      description:
+      - The name of the datacenter.
+      type: str
+      required: true
+      aliases: [ datacenter_name ]
+    enable_dpm:
+      description:
+      - Whether to enable DPM.
+      type: bool
+      default: false
+    default_dpm_behaviour:
+      description:
+      - Whether dpm should be automated or manual
+      type: str
+      default: automated
+      choices: automated, manual
+    host_power_action_rate:
+      description:
+      - specify host power action rate
+      type: int
+      default: 3
+      choices: [1, 2, 3, 4, 5]
+extends_documentation_fragment:
+- community.vmware.vmware.documentation
+
+'''
+
+EXAMPLES = r'''
+- name: Enable DPM
+  community.vmware.vmware_cluster_dpm:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    datacenter_name: datacenter
+    cluster_name: cluster
+    enable_dpm: true
+    default_dpm_behaviour: automated
+    host_power_action_rate: 2
+  delegate_to: localhost
```

Comment:
You can remove the `delegate_to: localhost` line from the example.

Reply:
I added a test

Reply:
@goneri There's a test playbook now. Is this PR now OK for you?

Reply:
Awesome, thank you!